src/lib/ts-lib-cache.ts

```diff
@@ -18,7 +18,8 @@ export async function loadDefaultLibMap(): Promise<Map<string, string>> {
   const missing: string[] = []
 
   for (const lib of libs) {
-    const cached = await get(CACHE_PREFIX + lib)
+    const cacheKey = `ts-lib-${TS_LIB_VERSION}-${lib}`
+    const cached = await get(cacheKey)
```

Comment:
The ttl check is missing for the typescript lib files, I think the TTL check should happen for this too!